src/plugins/clips-tf/clips_tf_thread.cpp

```diff
@@ -97,6 +98,15 @@ ClipsTFThread::clips_context_init(const std::string &env_name, LockPtr<CLIPS::En
 	  sigc::
 	    slot<CLIPS::Values, std::string, std::string, CLIPS::Values, CLIPS::Values, CLIPS::Values>(
 	      sigc::mem_fun(*this, &ClipsTFThread::clips_tf_transform_pose)));
+	clips->add_function("tf-add-publisher",
+	                    sigc::slot<CLIPS::Values, std::string>(
+	                      sigc::mem_fun(*this, &ClipsTFThread::clips_tf_add_publisher)));
+	clips->add_function("tf-publish-pose-static",
+	                    sigc::slot<CLIPS::Values, std::string, std::string, double, double, double>(
+	                      sigc::mem_fun(*this, &ClipsTFThread::clips_tf_publish_pose_static)));
+	clips->add_function("tf-publish-pose-non",
```

Comment:
this clips name should be changed to tf-publish-pose-non-static or just tf-publish-pose

Even better would be to just add a param that sets whether its static or not.